index.html

```diff
@@ -1,224 +1,299 @@
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
+      <div class="cards">
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
+      <div class="cards">
+        <div class="card-item">
+          <h1>Profesional Art Printing Data</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </p>
+          <ul class="skills-details">
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="project-details" type="button">See project</button>
+        </div>
+      </div>
     </section>
     <!-- ----------------------About section----------------->
-    <section class="about-section">
-        <div class="about-introduction">
-            <div class="about-introduction-content">
-                <h1 class="about-heading">About me</h1>
-                <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some
-                    of my work and experience! If you like what you see and have a project you need coded, don't
-                    hesitate to contact me.
-                </p>
-                <a href="https://www.linkedin.com/in/kifle-haile-5a613761/" target="_blank" rel="noopener noreferrer">
-                    <button class="my-resume-button" id="resume-button" type="button">Get My Resume</button>
-                </a>
-            </div>
-            <div class="background-graphic"></div>
+    <section id="about-section" class="about-section">
+      <div class="about-introduction">
+        <div class="about-introduction-content">
+          <h1 class="about-heading">About me</h1>
+          <p>
+            Hello I'm a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don't
+            hesitate to contact me.
+          </p>
```

Comment:
- [ ] Please make sure to limit the width of this paragraph as shown in the [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1278&t=t45uZqimwMP5BSMR-0) as yours seems to be a bit longer.
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/229134692-a706c47f-3eea-4dee-9a59-b12e0bd0bc3a.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/229134827-e3d65376-cc81-4c42-ad65-567531e11337.png)